Makefile

```diff
@@ -251,7 +244,8 @@ docker.exporter: build.exporter Dockerfile.exporter
 .PHONY: deps
 deps: header
 	@echo '--> Resolving dependencies...'
-	dep ensure
+	go mod vendor
+	go mod tidy 
```

Comment:
sequence can be wrong. tidy updates the module list and vendor will make copy of dependency. Better to run `go mod tidy` first then `go mod vendor`

Reply:
You may need to check go.mod/go.sum to verify dependency. Check  [this example](https://github.com/openebs/velero-plugin/blob/master/Makefile#L90)  for verification.